configs/configcat.json

```
@@ -1,7 +1,8 @@
 {
   "index_name": "configcat",
   "start_urls": [
-    "https://configcat.com/docs/"
+    { "url": "https://configcat.com/docs/", "selectors_key": "docsselector", "tags": [ "docs" ] }, 
+    { "url": "https://configcat.com/blog/", "selectors_key": "blogselector", "tags": [ "blog" ] },
```

```suggestion
    { "url": "https://configcat.com/docs/", "tags": [ "docs" ] }, 
    { "url": "https://configcat.com/blog/", "tags": [ "blog" ] }
```

If they both have the same structure (same selectors), they don't need a `selectors_key`. But if they are not similarly structured, you should see [this part of the doc](https://docsearch.algolia.com/docs/config-file/#using-custom-selectors-per-page)